zcash_transparent/src/bundle.rs

```diff
@@ -46,7 +50,7 @@ impl TransparentAuthorizingContext for EffectsOnly {
 pub struct Authorized;
 
 impl Authorization for Authorized {
-    type ScriptSig = Script;
+    type ScriptSig = script::Component<Opcode>;
```

Comment:
We have a similar problem here: it's possible to have a transaction with opcodes in a script_sig pre-v5, and therefore it's presumably possible to have a script_sig containing bad opcodes which are unused.